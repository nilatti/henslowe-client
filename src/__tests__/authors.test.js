// import Authors from '../Components/Authors/Authors';
// import renderer from 'react-test-renderer'
// import React from 'react';
// import Enzyme, { configure, mount, shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16'
// import MockRouter from 'react-mock-router';
//
// Enzyme.configure({ adapter: new Adapter() })
//
// test('authors component renders the authors div', () => {
//   const authors = [{id: 1, last_name: "mandigo"}]
//   const wrapper = shallow(<Authors authors={authors}/>)
//   expect(wrapper).toMatchSnapshot()
// });
