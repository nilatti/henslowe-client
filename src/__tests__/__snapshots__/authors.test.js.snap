// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authors component renders the authors div 1`] = `ShallowWrapper {}`;
